phonon/process.py

```diff
@@ -56,12 +57,13 @@ def __exit__(self, type, value, traceback):
     class AlreadyLocked(DisRefError):
         pass
 
-    def __init__(self, process_ttl=int(0.5*TTL), host='localhost', port=6379, db=1, heartbeat_interval=10, recover_failed_processes=True):
+    def __init__(self, process_ttl=int(0.5*TTL), host='localhost', hosts=None, port=6379, db=1, heartbeat_interval=10, recover_failed_processes=True):
```

Comment:
Would set the default to `hosts=['localhost']`, but this ensures backwards compatibility.